scripts/lib/CIME/SystemTests/erp.py

```diff
@@ -57,6 +57,10 @@ def _case_two_setup(self):
         self._case.set_value("CONTINUE_RUN", True)
         self._case.set_value("REST_OPTION","never")
 
+        # May need to change EXEROOT since separate_rundirs is False and separate_builds is True
```

Comment:
Would it work to put it in `_setup_cases`, either before or after the call to `_case_two_setup`?